app/views/hearings/show.html.haml

```diff
@@ -0,0 +1,75 @@
+= govuk_page_title(@hearing_day.strftime('%d/%m/%Y'), t('generic.hearing'))
+
+%table.govuk-table
+  %caption.govuk-table__caption.govuk-visually-hidden
+    = 'Hearing'
+  %tbody.govuk-table__body
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Hearing type
+      %td.govuk-table__cell
+        = @hearing.hearing_type
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Court
+      %td.govuk-table__cell
+        = @hearing.court_name || 'not set'
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Time listed
+      %td.govuk-table__cell
+        = @hearing_day&.strftime('%H:%M')
+
+%br
+.govuk-heading-m
+  = 'Attendees'
+
+%table.govuk-table
+  %caption.govuk-table__caption.govuk-visually-hidden
+    = 'Attendees'
+  %tbody.govuk-table__body
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
```

Comment:
As you don't have a table head, I suggest you change the first `td` of each row to a `th`, like `%th{ class: 'govuk-table__header', scope: 'row' }`.

The `th` font weight is bold, therefore you wont need  the class `govuk-!-font-weight-bold`

@jsugarman

Reply:
cool, good idea. I applied this to the defendants table as well